resources/assets/sass/base/_base.mixins.scss

```diff
@@ -0,0 +1,106 @@
+@mixin font-smoothing($value: antialiased) {
+
+	@if $value == antialiased {
+
+		-webkit-font-smoothing: antialiased;
+		-moz-osx-font-smoothing: grayscale;
+	} @else {
+
+		-webkit-font-smoothing: subpixel-antialiased;
+		-moz-osx-font-smoothing: auto;
+	}
+}
+
+@mixin text-rendering($value: optimizeLegibility) {
+
+	text-rendering: $value; // Joseph Pearson also points out that on slower machines (such as mobile devices) it can negatively impact page load when applied to large blocks of text.
+}
+
+@mixin vertical-center {
```

Comment:
As we use flexbox everywhere, can we replace it with flexbox vertical centering?